src/KurrentDB.SecondaryIndexing.Tests/Subscriptions/SecondaryIndexCheckpointTrackerTests.cs

```diff
@@ -290,5 +290,5 @@ public async Task Disposal_During_Active_Commit_Allows_Commit_To_Complete() {
 		Assert.True(commitCompleted);
 	}
 
-	private static readonly TimeSpan HalfOfASecond = TimeSpan.FromMilliseconds(500);
+	private static readonly TimeSpan FiveSeconds = TimeSpan.FromSeconds(5);
```

Comment:
Nit: It's better not to reflect the value as well as the type of the variable in its name. Just choose to `CommitSignalTimeout`